### ISSUE TITLE
Notification after completion of  Responsive Input Event (especially "custom" event types)

### ISSUE THREAD

**Issue by joanmarie** [w3c/editing#17]
In issue 14, I described a couple of scenarios in which the current set of "traditional" intentions might not cover everything we might wish it to. _If_ it is agreed that the intention event types should be expanded to include more and/or custom event types such as those outlined in issue 14, _then_ I think it's worth considering emitting notifications upon completion of those event types to address the following need described in the explainer:

```
Additionally, Accessibility Tools and other input modalities should be able to list available Intentions and understand what has occurred in order to provide feedback to users.
```

In the case of "traditional" intentions (caret moved, text changed, selection changed), user agents notify platform Assistive Technologies of the event via that platform's accessibility APIs. In the case of the "non-traditional" events like the next/previous tweet, user agents do similar notifications but the ATs are left trying to "understand what has occurred in order to provide feedback to users." For instance, in ATK/AT-SPI2, the following accessible events are seen when a user presses J to move to the next tweet in a Gecko-based browser:
1. keyboard event (J pressed)
2. object:property-change:accessible-name
3. object:property-change:accessible-name
4. object:text-changed:delete:system
5. object:children-changed:add:system
6. object:text-changed:insert:system
7. focus:
8. object:state-changed:focused
9. keyboard event (J released)

In other words, events which describe what happened within the user agent; not _why_ it happened (namely, the user intended to move to the next tweet and that movement occurred).

Having some sort of afterNextItem event would make things more readily and easily understood by ATs.

**Comment by johanneswilm** [comment]
As we don't cover more than pure contententering events as intentions, this does not apply. Please reopen if you still think it is of importance.
